docs/en/configuration/source-plugins/Hudi.md

```diff
@@ -0,0 +1,25 @@
+# Source plugin: Hudi
+
+## Description
+
+Read data from Hudi.
+
+## Options
+
+| name           | type   | required | default value |
+| -------------- | ------ | -------- | ------------- |
+| [hoodie.datasource.read.paths](#hoodiedatasourcereadpaths-string)           | string | yes      | -             |
+
+Refer to [hudi read options](https://hudi.apache.org/docs/configurations/#Read-Options) for configurations.
```

Comment:
We can list `hoodie.*` in Options and show the reference below.

Reply:
Updated, thx.